src/operation.rs

```diff
@@ -18,6 +18,7 @@ mod insert;
 pub(crate) mod list_collections;
 pub(crate) mod list_databases;
 mod list_indexes;
+#[cfg(feature = "in-use-encryption")]
```

Comment:
This is unrelated but the lack of it was causing dead code warnings when compiling without the feature selected.